Automattic/wp-calypso
# Jetpack Search: allow enabling for wp.com business sites

- allows end users to enable search on wp.com business sites - and any other allowed site
- still no upgrade message when not available

Testing:
- apply D17542 to sandbox - needed for the api changes
- go to settings => traffic for a site
- if it is eligible for JP Search (a8c sites only for now, but soon business sites too) then should be able to enable/disable the same as Jetpack Business or Atomic sites.
